Tests/DeepLearningTests/TensorTests.swift

```diff
@@ -0,0 +1,81 @@
+// Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import XCTest
+@testable import DeepLearning
+
+final class TensorTests: XCTestCase {
+    func testSimpleCond() {
+        func selectValue(_ pred: Bool) -> Tensor<Int32> {
+            let a = Tensor<Int32>(0)
+            let b = Tensor<Int32>(1)
+            if pred {
+                return a
+            }
+            return b
+        }
+
+        XCTAssertEqual(0, selectValue(true).scalar)
+    }
+
+    @inline(never)
```

Comment:
Could you help remove `@inline(never)` from all of these files? It's a GPE artifact that is no longer required.